adc/onboard_temperature/onboard_temperature.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
```

Comment:
Could you add a copyright header please?

Reply:
Like the other examples? Or should I put my name, email... ?

Reply:
I think as long as you license it under the same 3-Clause-BSD it's up to you whether you assign copyright over to Raspberry Pi or retain copyright yourself? But I'm not a legal expert :wink: